### ISSUE TITLE
Can't get it to work in a mushroom-chips-card

### ISSUE THREAD

**Issue by IzeQube** [daredoes/linked-lovelace-ui#34]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:** 2.2.0

**Last working release (if known):** -

**Browser and Operating System:**
I'm using Edge on Windows, but I don't think this has to do with it

**Description of problem:**
I'm seeing some progress with linked-lovelace so I am trying to move my templates from v1 to v2. I can't seem to be getting `ll_keys` working, but you signaled that you maybe want to remove it, so I'm trying to simplify my templates.

I've got a room card that consists of a `vertical-stack-in-card` that contains a `mushroom-template-card` that is working without a problem and a `mushroom-chips-card`, where I would like to use templates in the chips part.

With the configuration shown below, the my-room2 template is beeing applied without a problem, but the chip-templates are not used and ignored. But I have to use multiple chip-templates, as all my rooms have different (type of) sensors that I would like to use.

This is my card configuration that isn't working (ignore the `my-room2` part, thats working without a problem):
```yaml
type: custom:vertical-stack-in-card
cards:
  - ll_template: my-room2
    ll_context:
      name: Büro
      icon: fapro:duo-chair-office
      light_group: group.lights_buero
      entity_room_temperature: sensor.aqara_temp_buero_temperature
      entity_room_humidity: sensor.aqara_temp_buero_humidity
      entity_room_temphelper: binary_sensor.th_buro_offnen
      entity_room_trendhelper: sensor.th_buro_trend
      navigation_view: buero
  - type: custom:mushroom-chips-card
    chips:
      - ll_template: my-room-chip-light
        ll_context:
          entity: group.lights_buero
      - ll_template: my-room-chip-contact
        ll_context:
          entity: binary_sensor.fenster_buero
```

These is are my current templates for the `mushroom-chips-card`:
```yaml
type: light
ll_key: my-room-chip-light
entity: <%= context.entity %>
content_info: none
use_light_color: true
tap_action:
  action: toggle
double_tap_action:
  action: none
hold_action:
  action: none
```

```yaml
ll_key: my-room-chip-contact
type: conditional
conditions:
  - entity: <%= context.entity %>
    state: 'on'
chip:
  entity: <%= context.entity %>
  type: template
  icon: '{{ state_attr(''<%= context.entity %>'',''icon'') }}'
  icon_color: teal
  card_mod:
    style:
      .: |
        {% set num_on =
        expand(config.entity)
        |    
        selectattr('state', 'eq', 'on') | list | count %}
        {% if num_on > 1 %}
          ha-card:after {
            content: "{{ expand(config.entity) | selectattr('state', '==', 'on') | list | count }}";
            position: absolute;
            color: white;
            font-weight: normal;
            top: 0.8px;
            right: 4px;
            font-size: 8px;
            background-color: rgb(var(--rgb-warning));
            width: 12px;
            line-height: 12px;
            border-radius: 50%;
            text-align: center;
          }
        {% endif %}
```
**Javascript errors shown in the web inspector (if applicable):**
None

**Additional information:**

**Comment by daredoes** [comment]
@IzeQube to make sure I understand this right, as I think I know where the bug is.

The following block doesn't get transformed, but you expect it to for each one of the items in chips, right?

```
- type: custom:mushroom-chips-card
    chips:
      - ll_template: my-room-chip-light
        ll_context:
          entity: group.lights_buero
      - ll_template: my-room-chip-contact
        ll_context:
          entity: binary_sensor.fenster_buero

**Comment by IzeQube** [comment]
Yes, exactly.

**Comment by daredoes** [comment]
Okay, this issue is definitely in the code. 

I'll add your cards a test case and push a fix in `2.2.1` as soon as I can.

**Comment by daredoes** [comment]
@IzeQube new release is building now. I find the easiest way to trick HACS into showing the latest version is to click the redownload option, switch off of whatever version it has selected, and then you should be able to switch it to `2.2.1` and download the latest. You can always check your console logs for the version output as well.
